tools/Github/CreatePR.ps1

```diff
@@ -63,5 +65,24 @@ $Description
 * **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
 * **SHOULD NOT** adjust version of module manually in pull request
 "@
+
 $RequestBody = @{"title" = $Title; "body" = $PrBody; "head" = $HeadBranch; "base" = $BaseBranch }
-Invoke-WebRequest -Uri https://api.github.com/repos/Azure/azure-powershell/pulls -method POST -Headers $Headers -Body ($RequestBody | ConvertTo-Json)
\ No newline at end of file
+$Uri = "https://api.github.com/repos/Azure/azure-powershell/pulls"
+
+$PullRequests = Invoke-WebRequest -Uri $Uri -Headers $Headers | ConvertFrom-Json
+$ExistingPr = $PullRequests | Where-Object { $_.head.ref -eq $HeadBranch -and $_.base.ref -eq $BaseBranch }
```

Comment:
I mean pass these two parameters as filter to github server. Please follow this doc I sent to you: https://docs.github.com/en/rest/pulls/pulls?apiVersion=2022-11-28#list-pull-requests